### PR TITLE
Config map implementation

### PR DESCRIPTION
**Pull request by AjeyHiremath**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,8 +11,11 @@ ALL_BINARIES=$(PLUGIN_BINARY) $(DAEMON_BINARY)
 
 DAEMON_ACI_IMAGE=infoblox-cni-daemon.aci
 DAEMON_DOCKER_IMAGE=infoblox-cni-daemon
+INSTALL_DOCKER_IMAGE=tgogoi/infoblox-cni-install
```
Comment (adsri) [inline Makefile]: use infoblox docker hub

File: infoblox-cni-install.yaml [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
```
Comment (adsri) [inline infoblox-cni-install.yaml]: use kube-system namespace

File: install-cni.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```
Comment (adsri) [inline install-cni.sh]: can you please make sure volume mount donot require any permission.if require please try to fix that.Ex: as observed in openshift selinux issue.

File: install-cni.sh [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+
+set -e -x;
+
+if [ -w "/host/opt/cni/bin/" ]; then
+    cp -f /install/bin/* /host/opt/cni/bin/;
```
Comment (adsri) [inline install-cni.sh]: we are coping executable file  from this path to host path. Please make sure we are making it executable before copying.use chmod to make it executable.

File: infoblox-cni-install.yaml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+  name: kube-cni-install
+  labels:
+    tier: node
+    k8s-app: kube-cni-install
+spec:
+  selector:
+      matchLabels:
+        name: kube-cni-install
+  template:
+    metadata:
```
Comment (adsri) [inline infoblox-cni-install.yaml]: use kube-system namespace

File: infoblox-ipam.go [hunk 6]
```diff
@@ -38,7 +38,7 @@ type IBInfobloxDriver interface {
 	GetAddress(netviewName string, cidr string, ipAddr string, macAddr string) (*ibclient.FixedAddress, error)
 	UpdateAddress(fixedAddrRef string, macAddr string, vmID string) (*ibclient.FixedAddress, error)
 	ReleaseAddress(netviewName string, ipAddr string, macAddr string) (ref string, err error)
-	RequestNetwork(netconf NetConfig) (network string, err error)
+	RequestNetwork(netconf NetConfig, netview string) (network string, err error)
```
Comment (shravindran) [inline infoblox-ipam.go]: This change already taken care in my PR

File: install-cni.sh (hunk 3 above)
Comment (AjeyHiremath) [reply install-cni.sh]: will do this in next PR

File: Makefile (hunk 1 above)
Comment (AjeyHiremath) [reply Makefile]: will do it in next PR

File: infoblox-cni-install.yaml (hunk 2 above)
Comment (AjeyHiremath) [reply infoblox-cni-install.yaml]: Done

File: install-cni.sh (hunk 4 above)
Comment (AjeyHiremath) [reply install-cni.sh]: Done

File: Makefile (hunk 1 above)
Comment (AjeyHiremath) [reply Makefile]: Done

**Pull request merged**